easybuild/easyconfigs/g/GffCompare/GffCompare-0.10.6-foss-2017b.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GffCompare'
+version = '0.10.6'
+
+homepage = 'https://github.com/gpertea/%(namelower)s'
+description = """GffCompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'https://github.com/gpertea/%(namelower)s/archive/',
+    'https://github.com/gpertea/gclib/archive/',
+]
+
+gclib_version = '0.10.3'
```

Comment:
@HPC-UniOldenburg Any reason you're downloading an old version of `gclib` here? There is a `0.10.6` available too (now)...

We also need to be careful w.r.t. downloading from these two repositories, maybe we better put the source URL in `sources` in this particular case?

See also https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#common-easyconfig-param-sources-alt

Reply:
I stuck with the old version because otherwise both would be version 0.10.6 which caused huge problems during the fetching process, which I could not resolve with 'filename' and/or 'download-filename'. I Found another source file for gffcompare which contains the programs' name in the download file but I got problems during the building process when using this version combined with gclib 0.10.6, which I could not solve.
On the upside, the mentioned source file contains a folder with gclib, which makes the installation very easy but it is still version 0.10.3.
I hope this solution is okay to you.